zammad/values.yaml

```diff
@@ -101,23 +101,22 @@ zammadConfig:
     # Maximum upload size
     clientMaxBodySize: 50M
     knowledgeBaseUrl: ""
+    startupProbe:
+      tcpSocket:
+        port: 8080
+      failureThreshold: 10
```

Comment:
This would mean we have a 40s startup limit. Can we increase these `startupProbe.failureThreshold` values to `20` please (also in the other places below), just to be on the safe side? Then it would be similar as before.

Reply:
Apart from that, it's ready to merge IMHO.